src/app/screens/Home/index.tsx

```diff
@@ -59,23 +63,44 @@ function Home({ navigation }: Navigation) {
     if (!paginated.current && !!articles.length) paginated.current = true;
   }, [loading, paginated, articles]);
 
+  useEffect(() => {
+    if (currentTab === 1) {
+      getArticles();
+    }
+  }, [currentTab, getArticles]);
+
+  const handlePressTab = (index: number) => setCurrentTab(index);
+
   return (
     <SafeAreaView style={styles.container}>
-      <ScreenWithLoader
-        loading={loading && !articles.length}
-        refreshed={paginated.current}
-        withInitialLoading={false}>
-        <FlatList<Article>
-          style={styles.container}
-          data={articles}
-          keyExtractor={keyExtractor}
-          renderItem={renderItem}
-          ItemSeparatorComponent={renderSeparator}
-          onEndReachedThreshold={THRESHOLD}
-          showsVerticalScrollIndicator={false}
-          ListFooterComponent={renderFooter}
-        />
-      </ScreenWithLoader>
+      <CustomText center big darkBlue bold style={styles.title}>
+        {i18next.t('HOME:APP_NAME')}
+      </CustomText>
+      <TabList
```

Comment:
I think this tabList is not necessary when user is not authorized, like: 
`const currentUser = useSelector((state: State) => state.auth.currentUser);
{currentUser && (<TabList ..../>)}
`